prov/efa/src/rdm/efa_rdm_pke_rtr.c

```diff
@@ -66,6 +66,38 @@ ssize_t efa_rdm_pke_init_longcts_rtr(struct efa_rdm_pke *pkt_entry,
 	return 0;
 }
 
+/**
+ * @brief allcoate an RX entry for a incoming RTR packet
+ *
+ * The RX entry will be allocated from endpoint's OP entry
+ * pool
+ * @param[in]	pkt_entry	received RTR packet
+ *
+ * @return
+ * pointer to the newly allocated RX entry.
+ * NULL when OP entry pool has been exhausted.
+ */
+struct efa_rdm_ope *efa_rdm_pke_alloc_rtr_rxe(struct efa_rdm_pke *pkt_entry)
+{
+	struct efa_rdm_ep *ep = pkt_entry->ep;
+	struct efa_rdm_ope *rxe;
+	struct efa_rdm_rtr_hdr *rtr_hdr;
+
+	rxe = efa_rdm_ep_alloc_rxe(ep, pkt_entry->addr, ofi_op_read_rsp);
+	if (OFI_UNLIKELY(!rxe))
+		return NULL;
+
+	rxe->addr = pkt_entry->addr;
```

Comment:
This is also redundant, right? https://github.com/ofiwg/libfabric/blob/e08d3161b8038ebfa50a4eb5bf76132e4eaeb137/prov/efa/src/rdm/efa_rdm_ep_utils.c#L100

Reply:
Yep, I cleaned it for all `rta`, `rtr` and `rtw` in the new push